modules/module_loader/include/ModulesApp.h

```diff
@@ -135,8 +138,8 @@ ModulesApp::registerAllObjects(Factory & f, ActionFactory & af, Syntax & s)
   GeochemistryApp::registerAll(f, af, s);
 #endif
 
-#ifdef HEAT_CONDUCTION_ENABLED
-  HeatConductionApp::registerAll(f, af, s);
```

Comment:
Seems like if you are going to keep the above lines, then you should keep these as well

Reply:
this code is performed by inheritance we dont need to redefine it